src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -78,12 +88,28 @@ module.exports = {
   },
 
   /**
-   * This is currently a dummy function, but we may
-   * need it in the future to convert weird uris like
-   * `entity:node/27` to something resembling a
-   * relative url
+   * Takes an array meant to contain only one object.
+   * If this object exists, an object will be returned matching what is expected for "fieldLink" objects,
+   * otherwise null is returned.
+   *
+   * If the optional parameter is used, the returned object will only contain those attributes.
+   *
+   * @param {array} fieldLink
+   * @param {array} attrs
+   * @return {object}
    */
-  uriToUrl(uri) {
```

Comment:
Do we need to still export this function?

Reply:
Nope - with the refactor it is only used for the `createLink` function in the helper file:

![image](https://user-images.githubusercontent.com/2008881/71853513-87ccef00-3090-11ea-85a1-f2cf22dc52b8.png)